pyspecdata/lmfitdata.py

```diff
@@ -100,12 +99,6 @@ def functional_form(self, this_expr):
         )
         self.fit_axis = list(self.fit_axis)[0]
         # }}}
-        args = self.parameter_symbols + [str(*this_axis)]
-        self.fitfunc_multiarg = sp.lambdify(
-            args,
-            self.expression,
-            modules=[{"ImmutableMatrix": np.ndarray}, "numpy", "scipy"],
-        )
```

Comment:
I deleted this stuff because it's better to use `_v2`, then you brought it back.  Please leave it deleted.

It's not got to have more than one thing that's trying to do the same thing.

The new format uses the `variable_symbols` attribute, while this does not.

Please use `fitfunc_multiarg_v2` in your new additions, which should be possible.

Reply:
the ordering in `_v2` is different, and it's better to be consistent.  Also, we want to leave open the possibility that `variable_symbols` has more than one variable in it.  At the simplest level, you could just check that `variable_symbols` only had one element, and then feed `taxis` to the right argument of `multi...v2` -- for now it's OK to throw an error in the event that there is more than one variable.  (Down the line, we could just pass a dictionary to `taxis` if there is more than one.